Dockerfile

```diff
@@ -22,6 +22,9 @@ RUN yum update -y && yum install -y \
     # Dependencies for starting build as non-root user \
     sudo \
     \
+    # Dependiences for running Collect tests
+    openssh-server \
```

Comment:
This is needed for sftp-server so shared folders work, right? Maybe a comment more like "Dependencies for shared folder support"